### DISCUSSION TITLE
Co-accessibility for each cell type and compare the correlation between cell types

### DISCUSSION THREAD

**Discussion by ZunpengLiu** [GreenleafLab/ArchR#1790] [General]
Hello Ryan,

If I got right, from the full manual of ArchR, the co-accessibility analysis is based on all cell types [https://www.archrproject.com/bookdown/co-accessibility-with-archr.html](url). I am wondering how to compare the co-accessibility between different cell types or one cell type with distinct conditions. So, if we want to compare the co-accessibility between different cell types, we should subset the ArchR project and redo the`getCoAccessibility` analysis. Do I get it right?

And, I also have a question regarding the peak list used for calculating the co-accessibility, does ArchR re-call all peaks in this process? Can we specify the peak list used for co-accessibility analysis? For example, if we are interested in the co-accessibility difference for cell-type-specific peaks linked genes or the correlation difference between the cell-type-specific peak-gene links in different cell types, can we specify using the cell-type-specific peaks as input?

What's your suggestion for these analyses?

Thank you so much!

Best,

Zunpeng

**Comment by rcorces** [comment]
The following text is slated for incorporation into the manual with the next documentation release:

If you aim to only perform co-accessibility analyses on a subset of cells in your project, you can specify which cells should be analyzed using the `cellsToUse` parameter. However, it is worth noting that co-accessibility __requires__ variation. Since it is based on correlation, this type of analysis requires that some cells show co-accessibility while others do not. If you are looking to identify "cell type-specific" co-accessibility, you should not just limit your analysis to cells of that single cell type because you will squash the variability in the analysis. Instead, it is more effective to identify co-accessibility across all cells and overlap these co-accessibility predictions with a list of peaks of interest, for example cell type-specific peaks. Similarly, we don't think of co-accessibility as a metric that should be compared across cell types - we typically think of this as a framework for nominating putative gene regulatory interactions. While one could compare the difference in co-accessibility correlation for a given putative interaction across different groups of cells by using the `cellsToUse` parameter, it is important to note that correlation requires variation.


To answer your other question, it is the peak set in the `ArchRProject` that is used. You can change the peak set using the `addPeakSet()` function.

**Reply by ZunpengLiu** [reply]
Thank you so much for the prompt reply with so much helpful information.

I will conduct cell-type-specific analysis based on the co-accessibility across all cells and then integrate it with cell-type-specific peaks, that's a really great suggestion.

Many thanks!

Zunpeng
